protocol/x/vault/types/params.go

```diff
@@ -42,6 +42,10 @@ func (p QuotingParams) Validate() error {
 	if p.ActivationThresholdQuoteQuantums.Sign() < 0 {
 		return ErrInvalidActivationThresholdQuoteQuantums
 	}
+	// Skew factor times order_size_pct must be less than 2 to avoid skewing over the spread
+	if uint64(p.SkewFactorPpm)*uint64(p.OrderSizePctPpm) >= 2_000_000*1_000_000 {
```

Comment:
it might be better to use `big.Int` here to do multiplication + comparison. using uint64 won't overflow here with these two being uint32s, but in case we want to be future proof